sql/plan/create_user_data.go

```diff
@@ -117,3 +118,29 @@ func (a AuthenticationMysqlNativePassword) Password() string {
 func NewDefaultAuthentication(password string) Authentication {
 	return AuthenticationMysqlNativePassword(password)
 }
+
+// AuthenticationMysqlClearPassword is an authentication type that represents "mysql_clear_password".
+type AuthenticationMysqlClearPassword string
+
+// Plugin implements the interface Authentication.
+func (a AuthenticationMysqlClearPassword) Plugin() string {
+	return "mysql_clear_password"
+}
+
+// Password implements the interface Authentication.
+func (a AuthenticationMysqlClearPassword) Password() string {
+	return string(a)
+}
+
+// AuthenticationDoltJwt is an authentication type that represents "authentication_dolt_jwt".
+type AuthenticationDoltJwt string
+
+// Plugin implements the interface Authentication.
+func (a AuthenticationDoltJwt) Plugin() string {
+	return "authentication_dolt_jwt"
+}
+
+// Password implements the interface Authentication.
+func (a AuthenticationDoltJwt) Password() string {
+	return string(a)
+}
```

Comment:
Is this necessary? Is there a way to use the catalog plugins to not have the dolt-specific dependency here?